README.md

```diff
@@ -9,7 +9,16 @@ run
 ```
 go generate ./ent
 ```
-your html will be save at `ent/schema-viz.html`
+your html will be saved at `ent/schema-viz.html`
+# server via http
+You can use the helper function `entviz.Serve` to easily serve the static html page over http
+```golang
+h, err := entviz.Serve("./ent/schema", &gen.Config{})
+if err != nil {
+  panic(err)
+}
+http.ListenAndServe("localhost:3002", h)
+```
```

Comment:
Not sure this example is relevant after the new changes.

Reply:
correct, I accidentally put the fix in #4